openfisca_france/parameters/impot_revenu/calcul_reductions_impots/souscriptions/pme/souscription_capital/index.yaml

```
@@ -1,4 +1,11 @@
 description: Souscriptions au capital des PME
 metadata:
   short_label: Souscription de PME
+  order:
+  - seuil_tpe
+  - seuil
+  - taux
+  - taux18
+  - taux22
+  - taux25
   label_en: Small Firms Capital Subscription (PME) (1993 -)
```

```suggestion
  label_en: Small Firms Capital Subscription (PME)
```

Il faut retirer les intervalles de dates car ils sont reconstruits automatiquement désormais.